index.js

```diff
@@ -16,6 +16,7 @@ rulesConfiguration.addRule('di-unused', 0);
 rulesConfiguration.addRule('directive-name', 0);
 rulesConfiguration.addRule('directive-restrict', 0);
 rulesConfiguration.addRule('document-service', 2);
+rulesConfiguration.addRule('dumb-inject', 2);
```

Comment:
I think this should be changed to 0. We already have an issue about not enabling rules by default #258.